03 Writing Algorithms/03 Securities/99 Asset Classes/11 CFD/04 Market Hours/01 Introduction.html

```diff
@@ -1 +0,0 @@
-<p>This page shows the trading hours, holidays, time zone, and supported assets of the CFD market.</p>
```

Comment:
This PR shouldn't delete the "generic" [CFD > Market Hours](https://www.quantconnect.com/docs/v2/writing-algorithms/securities/asset-classes/cfd/market-hours) page